metricflow/model/objects/constraints/where.py

```diff
@@ -4,7 +4,7 @@
 import re
 from typing import List, Optional, Dict, Any
 
-from moz_sql_parser import parse as moz_parse
+from mo_sql_parsing import parse as mo_parse
```

Comment:
You need to update the pyproject.toml to reflect the dependency change and then run poetry update to reflect those changes in the poetry.lock file.

https://python-poetry.org/docs/basic-usage/#specifying-dependencies
https://python-poetry.org/docs/dependency-specification/
https://python-poetry.org/docs/cli/#update